src/Runner/index.js

```diff
@@ -80,6 +80,17 @@ class TestRunner {
     })
   }
 
+  /**
+   * Set the reporter function to a custom one
+   *
+   * @method reporter
+   *
+   * @private
+   */
+  reporter (reporterFn) {
+    this._reporter = reporterFn
```

Comment:
Can we add a check here that `reporterFn` is a function and not something else?

```
if (typeof (reporterFn) !== 'function') {
  throw new GE.InvalidArgumentException('')
}
```

Reply:
done